lib/sitemap/format_boost_calculator.rb

```diff
@@ -0,0 +1,20 @@
+class FormatBoostCalculator
+  FORMAT_BOOST_CONFIG = YAML.load_file('config/query/format_boosting.yml')
+
+  def initialize
+    @format_boosts = FORMAT_BOOST_CONFIG["format_boosts"]
+  end
+
+  def boost(format)
+    page_boost = @format_boosts[format] || 1
+
+    # Normalise format boost to always give a value between 0 and 1
+    page_boost / max_boost
```

Comment:
there is a small risk here that both values could be integers resulting in an incorrect result.

```page_boost.to_f / max_boost```